aptos-move/framework/supra-framework/sources/vesting_without_staking.move

```diff
@@ -287,6 +287,98 @@ module supra_framework::vesting_without_staking {
         }
     }
 
+    public entry fun create_vesting_contract_with_amounts (
+        admin: &signer,
+        shareholders: vector<address>,
+        amounts: vector<u64>,
+        schedule_numerator: vector<u64>,
+        schedule_denominator: vector<u64>,
+        start_timestamp_secs: u64,
+        period_duration: u64,
+        withdrawal_address: address,
+        contract_creation_seed: vector<u8>,
+    ) acquires AdminStore {
+        assert!(!system_addresses::is_reserved_address(withdrawal_address),
+            error::invalid_argument(EINVALID_WITHDRAWAL_ADDRESS),);
+        assert_account_is_registered_for_apt(withdrawal_address);
```

Comment:
@axiongsupra , create an issue to replace `assert_account_is_registered_for_apt` with `assert_account_is_registered_for_supra` everywhere in the code base and address it in a separate PR.

Reply:
see #90